examples/tadpole/downsample_source_images.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+from pathlib import Path
+
+from brainglobe_utils.IO.image import read_z_stack, save_any
+from loguru import logger
+
+from brainglobe_template_builder.preproc.transform_utils import (
+    downsample_anisotropic_image_stack,
+)
+
+
+def downsample_tadpole(folder: Path) -> None:
+    logger.info(f"Downsampling {folder.name}...")
+    stack = read_z_stack(str(folder))
+
+    in_plane_resolution = 1
+    out_of_plane_resolution = 3
+    in_plane_factor = int(target_isotropic_resolution / in_plane_resolution)
+    axial_factor = int(target_isotropic_resolution / out_of_plane_resolution)
+    downsampled = downsample_anisotropic_image_stack(
+        stack, in_plane_factor=in_plane_factor, axial_factor=axial_factor
+    )
+
+    sample_id = str(folder).split("_")[1].lower()
+    channel = "blue" if str(folder).split("_")[2] == "488" else "orange"
```

Comment:
I noticed they don't always use the absolute exact wavelength. Perhaps we should instead check if the wavelength is within ~10/20 nm of the target wavelength value for a given "color". But this is more for futureproofing, and may be left to future refactoring.

Reply:
Yea this is still tadpole-specific code anyway. Agree with the idea though.